content/W2015/W15-event-websitegitworkshop.md

```diff
@@ -0,0 +1,19 @@
+Title: Github/git and Website Workshop
+Date: 2015-03-1 12:00-17:00
+Category: Events
+Tags: workshop
+Slug: git-website-workshop
+Author: Evy Kassirer
+Summary: Come learn about web design and build your own website, hosted on Github.
+
+Interested in creating your own personal website? Women in Computer Science 
+will teach you the nuts and bolts of web design and how to use `git` through 
+GitHub! Join us for an afternoon of learning, coding, and snacking! Participants
+are expected to have basic knowledge of HTML and CSS.
+
+## Workshop Details ##
+
++ **Who:** Cis and trans women and non-binary folks
```

Comment:
You can now change this to `"[Women-only]({filename}/pages/faq.md)` or something similar now that #46's been merged :)